operator/src/main/resources/application.properties

```diff
@@ -126,15 +126,28 @@ topic=*;operations=describe;level=DEBUG \n\
 priority=1;topic=${managedkafka.canary.topic};operations=describe,read,write;level=DEBUG \n\
 group=*;apis=offset_fetch,offset_commit,heartbeat,describe_groups;level=DEBUG
 
+# default value for spec capacity - should be removed or replaced with a hard capacity
+managedkafka.kafka.max-connections=3000
+
+# topology settings
 managedkafka.kafka.colocate-with-zookeeper=true
 managedkafka.adminserver.colocate-with-zookeeper=true
 managedkafka.kafkaexporter.colocate-with-zookeeper=true
 managedkafka.canary.colocate-with-zookeeper=true
-managedkafka.kafka.max-connections=3000
+
+# sizing settings specific to OpenShift 4.8 on m5.2xlarge, and the topology settings
 managedkafka.kafka.container-cpu=4500m
 managedkafka.kafka.container-memory=19Gi
 managedkafka.kafka.jvm-xms=6442450944
 
+# absolute capacity specific to the above
+
+# this is meant as a pre-replication per broker value
+#
+# the spec capacity value is understood to be
+# X fully replicated partitions typically with replication factor 3
+managedkafka.kafka.partition-capacity=1500
+
```

Comment:
I'm not following what this property defines (versus the MK's `maxPartitions`). Is this saying that there can be 500 partitions with replication factor 3?

Reply:
It is supposed to be a physical limit supported per broker supported by this configuration / profile.

The MK maxPartitions is a user request for total partitions = maxPartitions * replication factor.

We use the physical limit to determine how many brokers are needed relative to that user request.